frontend/ui/test/widgets/trip_view_widget_test.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/trip_view_widget.dart';
+
+void main() {
+  testWidgets('Showing the text on page correctly shows up',
+      (WidgetTester tester) async {
+    var tripViewWidget = TripViewWidget();
+    await tester.pumpWidget(wrapForDirectionality(tripViewWidget));
+    await tester.pumpAndSettle();
+
+    // Should be everything on the view trip screen without being expanded...
+    expect(find.text('Trip Name'), findsOneWidget);
+    expect(find.text('M T D'), findsOneWidget);
```

Comment:
probably unnecessary but is there a way to test that the expanded view stuff is not on the page? or test that these elements and only these elements are on the page?

Reply:
yeah I could. Thanks for reminding because I definitely forgot to.